schema.sql

```diff
@@ -1,20 +1,51 @@
-/* Database schema to keep the structure of entire database. */
-
--- Table: public.animals
+CREATE TABLE IF NOT EXISTS owners
+(
+    id integer NOT NULL GENERATED ALWAYS AS IDENTITY ( INCREMENT 1 START 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 ),
+    name character varying COLLATE pg_catalog."default" NOT NULL,
+    CONSTRAINT owners_pkey PRIMARY KEY (id)
+);
```

Comment:
Great job @andym80 🤝

- [ ] As specified in the project requirements, kindly add the age column to the owners' table.
- [ ] You may also update your GitHub handles in your Readme file. kindly maintain one author and update your handles with clickable links. You can use [markdown syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) to achieve this.

Reply:
![Screenshot from 2022-10-21 21-40-17](https://user-images.githubusercontent.com/57109864/197308812-6e2730cc-a772-4074-b708-c62f0a494368.png)
Thanks, I'll update at PostgreSQL @1GORDON

Reply:
![Screenshot from 2022-10-21 22-01-35](https://user-images.githubusercontent.com/57109864/197309537-4c3f79f7-b6ef-4c08-9f97-37b82662760b.png)
@1GORDON